saleor/dashboard/views.py

```diff
@@ -65,6 +61,5 @@ def index(request):
 def get_low_stock_products():
     threshold = getattr(settings, 'LOW_STOCK_THRESHOLD', 10)
     products = Product.objects.filter(
-        Q(shirt__variants__stock__lte=threshold) |
-        Q(bag__variants__stock__lte=threshold)).distinct()
+        Q(variants__stock__quantity__lte=threshold)).distinct()
```

Comment:
This is more tricky as you should check only the highest stock level, not all of them. If one location contains 100 of the item, then the other only having 1 is not a reason to panic.